### PR TITLE
CF-510 Script to start VMs which are powered off.

### PR DESCRIPTION
**Pull request by plomakin**
Added script, whic is able to restore correct states of VMs
which are already present in the box.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry_devlab/cloudferry_devlab/tests/base.py [hunk 1]
```diff
@@ -426,20 +426,35 @@ def set_vm_state(novaclient, vm_id, vm_state):
...
         # emulate resize state:
         elif vm_state == u'resize':
-            novaclient.servers.resize(vm_id, '2')
+            try:
+                novaclient.servers.resize(vm_id, '2')
+            except nova_exceptions.Conflict:
+                pass
             return vm_id, ('VERIFY_RESIZE', 'ACTIVE', 'ERROR')
+        elif vm_state == u'active':
+            novaclient.servers.reset_state(server=vm_id, state=vm_state)
```
Comment (antonf) [inline cloudferry_devlab/cloudferry_devlab/tests/base.py]: Did you try to run functional tests against VMs started like this? Do they work?

Comment (plomakin) [inline cloudferry_devlab/cloudferry_devlab/tests/base.py]: nope, let me check that. maybe you're right and we should run servers.start()

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by antonf** [comment]
looks ok, +1

**Comment by plomakin** [comment]
still need some work to be done here, don't merge

**Comment by plomakin** [comment]
done

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by antonf** [comment]
👍

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**